extensions/amp-analytics/0.1/vendors.js

```diff
@@ -438,6 +438,35 @@ export const ANALYTICS_CONFIG = {
         '${actionParameter}${parameterSuffix}',
     },
   },
+
+  'shinystat': {
+    'requests': {
+      'ssurl': 'https://amp.shinystat.com/cgi-bin/getcod.cgi',
+      'ssparam': 'AMP=1' +
+                 '&USER=${account}' +
+                 '&AMPPAG=${page}' +
+                 '&AMPHR=${canonicalUrl}' +
+                 '&AMPREFER=${documentReferrer}' +
+                 '&AMPRES=${screenWidth}X${screenHeight}' +
+                 '&AMPCOLOR=${screenColorDepth}' +
+                 '&AMPCID=${clientId(AMP_CID)}' +
+                 '&AMPPID=${pageViewId}' +
+                 '&AMPTTL=${title}' +
+                 '&AMPRQC=${requestCount}' +
+                 '&AMPVIE=${viewer}' +
+                 '&AMPSSIDC=${queryParam(ssidc)}' +
+                 '&${ampp1}=${ampv1}' +
+                 '&${ampp2}=${ampv2}',
+      'pageview': '${ssurl}?${ssparam}',
+    },
+    'triggers': {
+      'pageview': {
+        'on': 'visible',
+        'request': 'pageview',
+        'iframePing': true,
```

Comment:
Yes, I'm aware of the stricter constraints of Safari. Anyway, I could not test this specific case. How can I simulate it? I used the "gulp serve" method as described in DEVELOPING.md to test the changes I submitted. But it does not explain how to make the document load from a (fake) CDN.